### PR TITLE
Update UI to display loaded Template

### PR DESCRIPTION
**Pull request by marvintxd**
UI now displays loaded `Template` after a successful `LoadTemplateCommand`.
`BrowserPanel` and `PersonListPanel` still exist in the MainWindow but are hidden.

Addresses #106.

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -34,17 +34,20 @@
     private Logic logic;
```
Comment (jhengy) [inline src/main/java/seedu/address/ui/MainWindow.java]: Removing PersonListPanel without modifying the system test may cause system tests to fail. Given the limited time, I chose to keep all the existing UI components and just "hide" it.

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -48,9 +48,9 @@
 
         <SplitPane id="splitPane" fx:id="splitPane" dividerPositions="0.4" VBox.vgrow="ALWAYS">
 
-          <VBox fx:id="personList" maxWidth="1">
-            <StackPane fx:id="personListPanelPlaceholder"/>
-          </VBox>
+          <!--<VBox fx:id="personList" maxWidth="1">-->
```
Comment (jhengy) [inline src/main/resources/view/MainWindow.fxml]: Note that setting maxWidth to 1 is as good as "hiding" it

File: src/main/java/seedu/address/logic/commands/LoadTemplateCommand.java [hunk 3]
```diff
@@ -20,36 +27,34 @@
         + "FILEPATH\n"
         + "Example: " + COMMAND_WORD + " "
         + "template1.txt";
-    public static final String MESSAGE_SUCCESS = "Attempting to load template from %1$s...";
-    //TODO: message_file_not_found
-    public static final String MESSAGE_NOT_FOUND = "The filepath is invalid";
+    public static final String MESSAGE_SUCCESS = "Successful load from %1$s.";
+    public static final String MESSAGE_NOT_FOUND = "Loading from %1$s failed.";
```
Comment (anubh-v) [inline src/main/java/seedu/address/logic/commands/LoadTemplateCommand.java]: Perhaps can change to name to `MESSAGE_TEMPLATE_NOT_FOUND`

File: src/main/java/seedu/address/logic/commands/LoadTemplateCommand.java [hunk 4]
```diff
@@ -20,36 +27,34 @@
...
-        //TODO: check for failure and return MESSAGE_NOT_FOUND
-        /*
-        Problem: unlike other commands (e.g. add, delete), Model does not know
-        if the load is successful. Since load raises an event, and the
-        StorageManager listens for that event to attempt to load.
-        */
         model.loadTemplate(filepath);
 
+        if (!isSuccessful) {
+            return new CommandResult(String.format(MESSAGE_NOT_FOUND, filepath));
```
Comment (anubh-v) [inline src/main/java/seedu/address/logic/commands/LoadTemplateCommand.java]: As discussed, will be good to check whether a CommandException is more appropriate

**Comment by marvintxd** [comment]
Resolves #106.

**Pull request merged**